timber/src/main/java/timber/log/Timber.java

```diff
@@ -382,6 +382,10 @@ protected boolean isLoggable(int priority) {
     }
 
     private void prepareLog(int priority, Throwable t, String message, Object... args) {
+
+      // consume tag, even if priority is not loggable, otherwise next message without explicit tag and loggable, will take erroneous tag
```

Comment:
This needs to wrap at 100 characters. It's why the build failed.

Please also capitalize the first letter and include a period. [Comments should be sentences.][1]

 [1]: nedbatchelder.com/blog/201401/comments_should_be_sentences.html